### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by gurdasnijor**
Adds a [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file to auto-assign reviewers across functional areas 

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# will be requested for review when someone opens a pull request.
+
+# Actions common lib folder
+actions-shared/ @segmentio/platform-components
+
+# AJV utils
+ajv-human-errors/ @segmentio/platform-components
+
+# Browser destinations
+browser-destinations/ @pooyaj
```
Comment (gurdasnijor) [inline .github/CODEOWNERS:12]: @poojavp95 - can you go ahead and create a github team to assign here?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence, these users
+# will be requested for review when someone opens a pull request.
+
+# Actions common lib folder
+actions-shared/ @segmentio/platform-components
```
Comment (mericsson) [inline .github/CODEOWNERS:6]:
```suggestion
actions-shared/ @segmentio/platform-components
```

should be build experience team. platform-components team is not active.

https://github.com/orgs/segmentio/teams/build-experience-team

File: .github/CODEOWNERS (hunk 1 above)
Comment (pooyaj) [reply .github/CODEOWNERS:12]:
```suggestion
browser-destinations/ @segmentio/libraries-web-team 
```

Comment (pooyaj) [reply .github/CODEOWNERS:12]: Done 💥

**Review by mericsson — APPROVED** [review]

**Pull request merged**